recipe/meta.yaml

```diff
@@ -8,37 +8,50 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 46d3fd2016cd5b842283d547877a02c418a0fe9aa7a6b0ae344115a2c990fef4
+  patches:
+    - patches/0001-rm-test-jsonpath-py.patch
+    - patches/0002-safe-eval.patch
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
+  source_files:
+    - test
   imports:
     - jsonpath
   requires:
+    - pip
     - pytest
-  source_files:
-    - test
   commands:
-    - pytest test/test1.py test/test2.py test/test3.py
+    - pip check
+    - pytest -v test/test1.py test/test2.py test/test3.py test/test4.py
```

Comment:
I'm assuming these tests are ran sequentially? Maybe we should use a loop or regex or something just in case they decide to make 100 tests? :sweat_smile:

Reply:
Yes, I don't like it -- and had to add the `test4.py` over and above the existing recipe.  However, `pytest test` refused to find anything so I've taken the simplest option.